src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/ConditionModalForm.jsx

```diff
@@ -27,12 +27,23 @@ class ConditionModalForm extends Component {
   }
 
   componentDidMount() {
-    const subflowOptions = this.props.subflows
-      .filter(flow => !flow.startsWith('skills/'))
-      .map(flow => ({
-        label: flow,
-        value: flow
-      }))
+    const getFlowLabel = name =>
+      name === 'main' ? 'Main' : name == 'error' ? 'Error handling' : name == 'timeout' ? 'Timeout' : name
+
+    const reorderFlows = flows =>
+      [
+        flows.find(x => x === 'main'),
+        flows.find(x => x === 'error'),
+        flows.find(x => x === 'timeout'),
+        ...flows.filter(x => !['main', 'error', 'timeout'].includes(x))
+      ].filter(x => Boolean(x))
```

Comment:
We also use the reordering / labeling in the flow builder, can you use the same logic here and on the flow builder ? This way we only need to edit it in one place.

Reply:
When I was coding, I knew there are three places in the code with same logic. But one in another module, and the one in flow builder makes it inline [here](https://github.com/botpress/botpress/blob/master/src/bp/ui-studio/src/web/views/FlowBuilder/sidePanel/util.tsx#L14), and ordering doesn't happen with string as here, but as Flow object [here](https://github.com/botpress/botpress/blob/master/src/bp/ui-studio/src/web/views/FlowBuilder/sidePanel/util.tsx#L67), so both places can't be used, unless we refactor them.

Reply:
We have some logic to share components and logic between the studio and modules. 
Example:

```js
// in any module
import { toastSuccess } from 'botpress/utils'
```
This is defined in `src/bp/ui-studio/src/web/components/Shared/Utils/index.tsx`
You could add a method called "reorderFlows" there.

(It's okay if you don't use the same logic for the label, since there is a different need for the tree view vs other places the name is required)